### ISSUE TITLE
Cannot install this package

### ISSUE THREAD

**Issue by MehulBawadia** [knuckleswtf/scribe#774]
### Scribe version

4.26.0

### PHP version

8.2

### Framework

Laravel

### Framework version

10.38.1

### Scribe config

```ruby
No config
```

### What happened?

I am installing this package using the steps provided in the docs.

Following are my composer.json `require` and `require-dev` details

```
"require": {
    "php": "^8.1",
    "guzzlehttp/guzzle": "^7.2",
    "laravel/framework": "^10.10",
    "laravel/sanctum": "^3.3",
    "laravel/tinker": "^2.8"
},
"require-dev": {
    "fakerphp/faker": "^1.9.1",
    "laravel/pint": "^1.0",
    "laravel/sail": "^1.18",
    "mockery/mockery": "^1.4.4",
    "nunomaduro/collision": "^7.0",
    "phpunit/phpunit": "^10.1",
    "spatie/laravel-ignition": "^2.0"
},
```

Now, when I run `composer require --dev knuckleswtf/scribe` I get the following error:

Error while installing without version
![Error while installing without version](https://github.com/knuckleswtf/scribe/assets/12674977/2eff5043-e743-4633-be40-d5de1b63f415)

Error while installing with the latest version - 4.26
![Error while installing with specific version](https://github.com/knuckleswtf/scribe/assets/12674977/3fef6daf-31f4-4e10-9c5e-ee652cb7e62d)

This project I am trying to install it on is already up-to-date. Could please you help?

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by shalvah** [comment]
Based on the second-to-last bullet point (please post text next time, not an image), it appears you have symfony/yaml v7 installed, and I think we don't support it yet (it came out last month).
Can you verify? (`composer show symfony/yaml`)

**Comment by shalvah** [comment]
I've relaxed the version constraint to allow Symfony v7. Can't test it because some of our dev dependencies still require v6. You can test it out by requiring `dev-master` of Scribe.

**Comment by shalvah** [comment]
Released in 4.27.0
